src/Nav/index.js

```diff
@@ -12,32 +12,45 @@ governing permissions and limitations under the License.
 import React from 'react'
 import PropTypes from 'prop-types'
 import { withPrefix } from 'gatsby'
-import { defaultFocus } from '../ManifestUtils'
 import { isExternal } from '../utils'
 
 import '@spectrum-css/sidenav'
 import './index.css'
 
-const nav = (data, defaultFocus, currDepth, maxDepth) => {
+const isPathSelected = (path, selected) => {
+  const compare = !selected.endsWith('/')
+    ? selected
+    : selected.substring(0, selected.length - 1)
+  return path === compare
+}
+
+const nav = (data, selected, currDepth, maxDepth) => {
   if (currDepth >= maxDepth) {
     return ''
   }
 
   return (
     <ul className='spectrum-SideNav  spectrum-SideNav--multiLevel'>
       {data.map((node, index) => {
-        const isSelected =
-          node.title === defaultFocus
-            ? 'spectrum-SideNav-item is-selected'
-            : 'spectrum-SideNav-item'
         const path = isExternal(node.path) ? node.path : withPrefix(node.path)
+        const isSelected = isPathSelected(path, selected)
+          ? 'spectrum-SideNav-item is-selected'
+          : 'spectrum-SideNav-item'
         return (
           <li className={isSelected} key={index}>
-            <a href={path} className='spectrum-SideNav-itemLink'>
+            <a
+              href={path}
```

Comment:
this changes behavior so not sure if it's a good idea: would it make sense to add `target="_blank"` here?

Reply:
We don't want target="_blank" as that would open each page in the side nav in a new tab which seems a bit excessive IMHO.